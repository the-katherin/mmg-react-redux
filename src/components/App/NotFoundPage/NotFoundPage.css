.not-found-window {
  display: grid;
  justify-content: center;
  grid-row-gap: 20px; 
  padding: 30px;
}

.not-found-window__button {
    justify-self: center;
}