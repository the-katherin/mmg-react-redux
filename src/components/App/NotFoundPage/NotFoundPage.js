import React from 'react';
import { Link } from "react-router-dom";
import './NotFoundPage.css';

const NotFoundPage = () => (

    <div className="not-found-window">

        <h2>Unfortenately, the page you're looking for isn't found :(</h2>

        <Link to='/form' className='not-found-window__button'>
            <button >
                Got to form selection window
             </button>
        </Link>
    </div>
);

export default NotFoundPage;