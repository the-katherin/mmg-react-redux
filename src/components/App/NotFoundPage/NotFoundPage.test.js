import React from "react";
import NotFoundPage from "./NotFoundPage";

let wrapper;

describe("NotFoundPage", () => {

    console.log(NotFoundPage);

    beforeEach(() => {
        wrapper = shallow(<NotFoundPage />)
    });

    test("match snapshot", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    test('redirection button exists', () => {
        const button = wrapper.find('button');
        expect(button).toBeDefined();
    });

});
