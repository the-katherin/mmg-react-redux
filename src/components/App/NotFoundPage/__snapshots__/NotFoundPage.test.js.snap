// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage match snapshot 1`] = `
<div
  className="not-found-window"
>
  <h2>
    Unfortenately, the page you're looking for isn't found :(
  </h2>
  <Link
    className="not-found-window__button"
    replace={false}
    to="/form"
  >
    <button>
      Got to form selection window
    </button>
  </Link>
</div>
`;
